src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -85,10 +86,9 @@ public static Address parseAddress(String address) throws ParseException {
      * Parses a {@code String note} into a {@code Note}.
      * Leading and trailing whitespaces will be trimmed.
      */
```

Comment:
We should update the javadocs as this function no longer takes in a `String`
```suggestion
     * Parses a {@code Optional<String> note} into a {@code Note}.
     * Leading and trailing whitespaces will be trimmed.
     * If the {@code Optional} is empty, return a {@code Note} with an empty string.
     */
```

Reply:
Thanks for pointing that out!